articles/defender-for-iot/organizations/traffic-mirroring/configure-mirror-hyper-v.md

```diff
@@ -139,6 +163,24 @@ Get-VMSwitchExtensionPortFeature -FeatureName "Ethernet Switch Port Security Set
 |--|--|
 |**vSwitch_Span** | Newly added SPAN virtual switch name |
 
+## Configure VLAN settings for the Monitor adapter (if needed)
+
+In case the HYper-V server would sit in a diffeernt VLAN than the VLAN from which the mirrored traffic comes, the Monitor adapter must be set to accpet traffic from the mirrored VLANs
+
+Use below PowerShell command to enable the Monitor adapter to accept traffic from the monitored traffic from different VLANs:
```

Comment:
If the Hyper-V server is located in a different VLAN than the VLAN from which the mirrored traffic originates, set the monitor adapter to accept traffic from the mirrored VLANs.

Use this PowerShell command to enable the monitor adapter to accept the monitored traffic from different VLANs: